src/elements/e-picture.vue

```diff
@@ -163,10 +165,16 @@
          */
         loaded: false,
 
+        /**
+         * @type {String} Holds srcset string of comma separated sources with width value.
+         */
+        src: this.srcset,
```

Comment:
todo: please use `internalSrcSet` as key name, since `src` is also an `img` related attribute.